portalnet/src/overlay_service.rs

```diff
@@ -184,6 +186,9 @@ pub enum OverlayRequestError {
 
     #[error("Received invalid remote discv5 packet")]
     InvalidRemoteDiscv5Packet,
+
+    #[error("Content wasn't found on the network")]
+    ContentNotFound((String, bool, Option<QueryTrace>)),
```

Comment:
Can you document what parameters are? Maybe doing something like this:
```
ContentNotFound {
  message: String,
  utp_transfer: bool,
  trace: Option<QueryTrace>,
}
```
You can also add `message` to the error string.